gradle/libs.versions.toml

```diff
@@ -145,6 +147,7 @@ httpclient5 = { group = "org.apache.httpcomponents.client5", name = "httpclient5
 mockserver-netty = { group = "org.mock-server", name = "mockserver-netty", version.ref = "mockserver" }
 mockserver-client-java = { group = "org.mock-server", name = "mockserver-client-java", version.ref = "mockserver" }
 commons-lang3 = { group = "org.apache.commons", name = "commons-lang3", version.ref = "commons-lang3" }
+commons-lang = { group = "commons-lang", name = "commons-lang", version.ref = "commons-lang" }
```

Comment:
Why do we need this? Because we have used lang3.

Reply:
`Hadoop-aws` with version 3.1.0 depends on `commons-lang`, which is brought by `hadoop-common` in runtime environment.

In test environment, the version of `hadoop-common` is upgraded from 3.1.0 to 3.3.6 because of conflict with `hadoop-minicluster`, which removes the dependency of `commons-lang`. So we have to add `commons-lang` manually to run tests.

Reply:
Maybe you use `hadoop-aws` 3.3.6 in the test.

Reply:
I agree, I think we should upgrade the Hadoop version to a newer one to handle this.